app/src/main/java/de/westnordost/streetcomplete/quests/ImageListQuestAnswerFragment.java

```diff
@@ -85,7 +85,7 @@
 
 	protected int getItemsPerRow()
 	{
-		return 4;
+		return Math.min(4, getItems().length);
```

Comment:
This and the changes to the other quests is not a good idea. The quest types below have 3 per row, not because they are exactly three choices, but the size of photo-selection-items should be not still smaller than that. Pictograms and icons are usually clear enough that they can be smaller.

Reply:
OK, I will drop this part.

> Pictograms and icons are usually clear enough that they can be smaller.

Is it still desirable to show two small icons with blank space on the right side (that is what happens where there is not enough entries to fill even the first row?

Generally, I see not even single quest where getItemsPerRow was set to value higher than count of possible answers.

Reply:
removed from this PR